tests/robot-tests/tests/general_public/prod_table_tool.robot

```diff
@@ -41,70 +41,70 @@ check publications have correct number of releases
 
 check 'Children's social care' contains correct topics
     user expands details section    Children's social care
```

Comment:
There is already a keyword to open a details dropdown which I think can be used here: `user opens details dropdown`. This is preferable as it actually waits for the dropdown to expand, which means  you shouldn't have to add waits to the `user checks page for details dropdown` keywords after it. So I would remove all those waits.

It seems like `user expands details section` is only being used in this test suite, so after subsituting all instances with `user opens details dropdown`, I think you can remove `user expands details section` from `common.robot`.

Reply:
Cool, didn't realize this was in common.robot. I've removed it